pkg/internal/discover/finder.go

```diff
@@ -77,11 +76,10 @@ func (pf *ProcessFinder) Start(cfg *beyla.Config) (<-chan *ebpf.ProcessTracer, <
 func newGoTracersGroup(cfg *beyla.Config, metrics imetrics.Reporter) []ebpf.Tracer {
 	// Each program is an eBPF source: net/http, grpc...
 	return []ebpf.Tracer{
-		nethttp.New(&cfg.EBPF, metrics),
-		&nethttp.GinTracer{Tracer: *nethttp.New(&cfg.EBPF, metrics)},
-		grpc.New(&cfg.EBPF, metrics),
-		goruntime.New(&cfg.EBPF, metrics),
-		gosql.New(&cfg.EBPF, metrics),
```

Comment:
Did you forget to re-add gosql?

Reply:
I removed it temporarily, the whole separate file and EBPF program, because it required more headers refactor in the BPF side, so we don't include collections we don't need twice. I'm going to follow-up with a PR to split that off again after I clean up the headers more.